### PR TITLE
added DaemonSets

### PR DESCRIPTION
**Pull request by killabayte**
- added DaemonSets

### CODE REVIEW

**Comment by Arriven** [comment]
@killabayte still conflicts

File: k8s-manifest/advanced/01_db1000n_daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    matchLabels:
+      name: db1000n
+  template:
+    metadata:
+      labels:
+        name: db1000n
+    spec:
+      containers:
+      - name: db1000n
+        image: oneodinscrow/db1000n-light
```
Comment (Amet13) [inline k8s-manifest/advanced/01_db1000n_daemonset.yaml:19]: Is that correct image?

Comment (killabayte) [reply k8s-manifest/advanced/01_db1000n_daemonset.yaml]: I'll change to the original one, that is my personal repo, where I run test

**Comment by killabayte** [comment]
Guys, I can't see what exactly I need to fix to resolve those conflicts. It would be super great if you can point me to the particular piece that should be fixed.

File: README.md [hunk 2]
```diff
@@ -172,12 +172,14 @@ the command above will detect the os and architecture, dowload the archive, vali
 ./db1000n
 ```
 
-### k8s run
+### Kubernetes run
 
-Enter k8s directory
+## Simple run with deployment
+
+Enter to k8s simple deployment directory
```
Comment (Arriven) [inline README.md:179]: I'll merge it anyway but I think it should be "Enter k8s simple deployment directory", someone good at English should confirm that

**Review by Arriven — APPROVED** [review]

**Comment by Arriven** [comment]
hmm, ok, github mobile client was showing me that it's possible to merge this. Guess I'll have to cherry-pick it manually. @killabayte the easiest way for you to bring your fork back up to date after that would be to delete and re-fork the repo (sorry for that) (and please wait until I reply here that I've applied your patches)

**Comment by Arriven** [comment]
@killabayte I cherry-picked commits manually, you should be good to go recreate your fork

**Pull request closed without merging**